meetings/2022-01-10.md

```diff
@@ -0,0 +1,144 @@
+* Nicolas AS
+* Tom Haegemans
+* Wouter Termont
+* Matthieu Bosquet
+* Elf Pavlik
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Use panel chat and repository. Queue in call to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md)
+* If this is your first time, welcome! please introduce yourself.
+
+### Scribes
+
+* Nicolas AS
+
+---
+
+## Agenda
+
+### Pull Requests
+https://github.com/solid/solid-oidc/pulls
+
+* [Add RS associated AS issuing Access Token PR#18](https://github.com/solid/solid-oidc/pull/18)
+
+### Issues
+https://github.com/solid/solid-oidc/issues
+
+* [Organize sections by conformance classes I#65](https://github.com/solid/solid-oidc/issues/65)
+
+
+## Discussions
+
+### Short introductions
+
+* NAS: I'm a developer at Inrupt, working on the SDK
+* AC: I'm also developer at Inrupt, working on the enterprise server
+* TH: I'm a cofounder of Digita. We have questions about UMA
+
+### First public working draft timeline
+
+* MB: What is the timeline for the FWPD ?
+* AC: We hope the FPWD to be published within the month (not a hard commitment though).
+The remaining items to be sorted are: 
+    * [Solid-OIDC PR#18](https://github.com/solid/solid-oidc/pull/18)
+    * Reorganizing the sections by conformances classes
+    * Setting up automation and consistency check
+* MB: Should we take action on updating the primer is updated ?
+* AC: Definitely, any help is appreciated.
+* EP: The primer is updated as part of PR#18 
+
+#### Digita question about UMA
+
+* TH: UMA has been written without assumptions about the relationship between the access
+control and the actual resources. In UMA, the access controls can be kept in the AS domain,
+while in solid they may be stored in the RS, which feels perpendicular.
+* EP: These are excellent questions. This Wednesday in the AuthZ and interop panel will be
+dedicated to this. One of the assumptions in Solid-OIDC is that there's a shared security
+domain between the RS and the AS, so how they sync is out of scope of the spec. They synchronize
+to have a single source of truth. The AS can generate ACRs based on ??? in the interop spec.
+* AC: Let me first comment on OIDC/Oauth2. OIDC descibes how an ID token can be generated,
+access tokens are out of scope except for the userinfo enpoint. The fact that we've been
```

Comment:
```suggestion
access tokens are out of scope except for the user info endpoint. The fact that we've been
```

Reply:
OpenID Connect uses a single word for `userinfo`: https://openid.net/specs/openid-connect-core-1_0.html#UserInfo

Reply:
@acoburn -- Adjusted requested change to suit. Thanks!